modules/javafx.controls/src/test/java/test/javafx/scene/control/TableViewTest.java

```diff
@@ -798,6 +798,37 @@ public void ensureRowRemainsSelectedWhenSelectingCellInSameRow() {
     /*********************************************************************
      * Tests for specific bugs                                           *
      ********************************************************************/
+    @Test public void test_jdk_8277853() {
```

Comment:
Not sure if there is some preference around here but I always like if the test method is not just named after the ticket but instead what it actually tests, e.g. something like `testInvisibleScrollbarDoesNotScrollTableToBeginning`. Optionally the ticket number can be referenced in the javadoc then. But that's just me.

Reply:
I almost made the same comment as you did. In general we no longer use the pattern of putting the bug ID in the test name. In this case all of the surrounding tests do that, so I decided not to mention it, but I am perfectly happy for the test name to be changed to reflect its function (with the bug ID in a comment).